apps/epic-react/src/pages/invoices/index.tsx

```diff
@@ -23,7 +23,11 @@ export const getServerSideProps: GetServerSideProps = async ({req}) => {
     const purchases = await getPurchasesForUser(sessionToken?.sub)
     return {
       props: {
-        purchases: purchases.map(convertToSerializeForNextResponse),
+        purchases: purchases
+          .filter((purchase: any) =>
+            purchase.product.name.includes('Epic React'),
```

Comment:
all of the products have this in the title so the approach works and sucks a little less than `products[0]` but not much...